src/main/java/org/fcrepo/migration/ObjectReference.java

```diff
@@ -34,12 +34,6 @@
      */
     public ObjectProperties getObjectProperties();
 
-    /**
-     * Lists all datastream ids.
-     * @return all datastream ids
-     */
-    public List<String> listDatastreamIds();
```

Comment:
Why was this removed?

Reply:
it's unused (like isLastVersionIn). Should I put it back in as well?

Reply:
Yeah, I think so. While I do agree with your eagerness to cleanup the cruft, with this particular project I'd be more inclined to leave well enough alone so long as it's not actively harmful.